### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by mattumland**
## Is this a fix or a feature?
Fix
## What does it fix?
Removes a host of small errors
## How should this be tested?
Run dev server and check for errors

### CODE REVIEW

File: src/components/Modal.vue [hunk 1]
```diff
@@ -36,13 +35,17 @@
               <button v-else name="remove" v-on:click="emitWatchlist(false)">ⓧ Watchlist</button>
             </aside>
           </section>
+          <div v-if="movie.backdrop.includes('null')">
+              <img class="backup-poster" :src="movie.poster" alt="movie poster">
```
Comment (Jacksonmcguire) [inline src/components/Modal.vue:39]: good thought here

File: src/components/Modal.vue [hunk 2]
```diff
@@ -36,13 +35,17 @@
...
+          </div>
```
Comment (jgiwinski) [inline src/components/Modal.vue:40]: wow whoever got this working is a genius....

File: src/components/Modal.vue [hunk 3]
```diff
@@ -224,6 +229,17 @@ export default {
 .detail-splash {
   background-size: cover;
   background-repeat: no-repeat;
+  background-color: $gray;
+}
+
+.modal__dialog::-webkit-scrollbar {
+    background-color: transparent;
+  }
```
Comment (jgiwinski) [inline src/components/Modal.vue:237]: also a genius to the big brain who fixed this

File: src/components/Modal.vue [hunk 4]
```diff
@@ -244,7 +260,7 @@ export default {
 border: 2px solid;
 border-radius: 50%;
 height: 50px;
-width: 50px;
+min-width: 50px;
```
Comment (jgiwinski) [inline src/components/Modal.vue:263]: glad this fixed this smashed rating circle.

**Review by Jacksonmcguire — APPROVED** [review]
I am Jackson McGuire and this has been approved

**Review by jgiwinski — APPROVED** [review]
This is Julia Iwinski and I approve this commit.

**Pull request merged**